README.md

```diff
@@ -1,2 +1,247 @@
-# books-store
-this is books store react project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
### [OPTIONAL]

- [ ] Please fix the Micorverse logo image in the README file 👍🏻 
<img width="955" alt="Screenshot 2022-11-19 at 11 59 21 AM" src="https://user-images.githubusercontent.com/83048208/202838953-a2a1d258-6a2e-4b7b-bc02-9c93dbfcfecc.png">